partner_multi_relation/models/res_partner_relation.py

```diff
@@ -152,6 +159,13 @@ def _check_relation_uniqueness(self):
                     ("date_start", "<=", record.date_end),
                 ]
             if record.search(domain):
-                raise ValidationError(
-                    _("There is already a similar relation with " "overlapping dates")
+                partner_message = _(
+                    "%(left_partner)s already has a %(connection_type)s"
+                    " relation with %(right_partner)s with overlapping dates"
                 )
+                message = partner_message % {
```

Comment:
Translation function interprets in >= v16 args as arguments for templating. Please use them instead.